### DISCUSSION TITLE
Option to set the first day of the week in datapicker

### DISCUSSION THREAD

**Discussion by igiornotigi** [directus/directus#18703] [Feature Requests]
### Summary

Now the data picker in the date interface shows Sunday as the first day of the week.
It would be nice to have an option to start it on a Monday

### Motivation

In many countries the week starts on a Monday

### Detailed Design

Insert a global option on general project settings and a local option in the interface settings (to override locally)

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

no idea

### Alternatives

no idea

### Adoption Strategy

no idea

**Comment by JPustkuchen** [comment]
Looks like a duplicate of https://github.com/directus/directus/discussions/18703?

**Reply by rijkvanzanten** [reply]
This... is #18703 👀

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
